circuits/cpp/src/aztec3/circuits/rollup/merge/.test.cpp

```diff
@@ -1,12 +1,30 @@
 #include <gtest/gtest-death-test.h>
 #include <gtest/gtest.h>
 #include "aztec3/circuits/rollup/merge/init.hpp"
-#include "aztec3/circuits/rollup/merge/utils.hpp"
-#include "aztec3/circuits/rollup/merge/native_merge_rollup_circuit.hpp"
-#include "aztec3/utils/dummy_composer.hpp"
 #include "c_bind.h"
+#include "aztec3/circuits/rollup/test_utils/utils.hpp"
+#include "index.hpp"
+#include "init.hpp"
 
-namespace aztec3::circuits::rollup::merge {
+namespace {
+using aztec3::circuits::abis::BaseOrMergeRollupPublicInputs;
```

Comment:
I think these don't confirm to the new specs that @dbanks12 proposed

Reply:
Think this can be addressed separately? We would probably need to handle it for more than just the changes in here.